applications/frontend/src/types/model.ts

```diff
@@ -0,0 +1,10 @@
+import { FeaturedType } from './global'
+
+// Model object
+export interface Model {
```

Comment:
inconsistent linting. Add eslint --fix to the build script?

Reply:
?

Reply:
Other files have 2 spaces, here 4